src/pages/api/v1/user/interview-prep/sheet.ts

```diff
@@ -62,6 +67,17 @@ const handleMarkQuestionCompleted = async (
       );
     }
 
+    // Convert isCompleted to a proper boolean
+    const isCompletedBool = isCompleted === true || isCompleted === 'true';
```

Comment:
isCompleted SHOULD always be Boolean. There is no other way. If NOT fix that. Don't find the workaround